src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx

```diff
@@ -30,7 +30,7 @@ const FundingPeriod = () => {
   const ssvStore: SsvStore = stores.SSV;
   const processStore: ProcessStore = stores.Process;
   const operatorStore: OperatorStore = stores.Operator;
-  const [customPeriod, setCustomPeriod] = useState(1);
+  const [customPeriod, setCustomPeriod] = useState(730);
```

Comment:
Move custom period value to constants (when I say constants i mean config.ts file)

Reply:
done